bnd-workspace/de.fraunhofer.abm.app/web/controllers/LoginController.js

```diff
@@ -51,7 +51,7 @@ function($rootScope, $scope, $http, $location, $cookies, ngCart, GoogleSignin, N
 							Notification.error('Wrong username or password');
 							$location.path('/login');
 						} else {
-							Notification.error('Failed with ['+ d.status + '] '+ d.statusText);
+							Notification.error('Login Failed due to some internal error. Please try agin later');
```

Comment:
Correct the English: capital letters only at the start of a sentence.
 Also correct in other places.